### PR TITLE
Attach label to namespace, to aid in owner identification

### PR DESCRIPTION
**Pull request by abandali**
Extended the `create-namespace` command to slap on a label/tag with the id (from AWS) of the person that created the namespace; this will make it easier when removing potentially orphaned namespaces

### CODE REVIEW

File: aladdin/bash/container/add-aws-assume-role-config/add-aws-assume-role-config [hunk 1]
```diff
@@ -59,7 +59,7 @@ function usage {
 	EOF
 }
 
-if [[ $# -eq 0 || "$1" == "-h" || "$1" == "--help" ]]; then
+if [[ $# -lt 4 || "$1" == "-h" || "$1" == "--help" ]]; then
```
Comment (abandali) [inline aladdin/bash/container/add-aws-assume-role-config/add-aws-assume-role-config:62]: scope creep 😱

File: aladdin/bash/container/create-namespace/create-namespace [hunk 2]
```diff
@@ -19,6 +23,9 @@ function usage {
 
 if [[ $# -eq 0 || "$1" == "-h" || "$1" == "--help" ]]; then
     usage
+elif [[ $# -eq 1 ]]; then
+	owner="$(aws sts get-caller-identity --query 'UserId' --output text | awk -F ':' '{print$2}')"
```
Comment (abandali) [inline aladdin/bash/container/create-namespace/create-namespace:27]: I prefer `--query` over `| jq` even though it requires an extra param `--output text`

**Comment by abandali** [comment]
both feedback items incorporated, ran it on KDEV:
```
$ aladdin -c KDEV create-namespace bozo
...
namespace/bozo created
namespace/bozo labeled

KDEV1115:abbas> kubectl describe namespace bozo
Name:         bozo
Labels:       owner=abbas.bandali.ops
Annotations:  <none>
Status:       Active
```

File: aladdin/bash/container/create-namespace/create-namespace (hunk 2 above)
Comment (jarojasm95) [reply aladdin/bash/container/create-namespace/create-namespace:27]: what is the value of `owner` if the `aws` command fails? does it make the whole script fail?

File: aladdin/bash/container/create-namespace/create-namespace [hunk 3]
```diff
@@ -3,14 +3,18 @@ set -eu -o pipefail
 
 function create_namespace {
     kubectl create namespace $1
+    kubectl label --overwrite namespace $1 owner=$2
```
Comment (jarojasm95) [inline aladdin/bash/container/create-namespace/create-namespace:6]: maybe ignore if there's an error here? (like invalid characters in value, or something of that nature)
```suggestion
    kubectl label --overwrite namespace $1 owner=$2 ||:
```
since it's a non-critical piece of functionality (I think)

**Review by jarojasm95 — COMMENTED** [review]
mostly looks good!

File: aladdin/bash/container/create-namespace/create-namespace (hunk 2 above)
Comment (abandali) [reply aladdin/bash/container/create-namespace/create-namespace]: > what is the value of `owner` if the `aws` command fails? does it make the whole script fail?

the script will fail - I think we have this flaw in several places in aladdin - I went ahead and plumbed in a default owner of `nobody` so that the script will not fail and complete gracefuilly:
```
IT2771:aladdin Abbas.Bandali$ aladdin -c KDEV create-namespace bozo
...
/root/aladdin/aladdin/bash/container/create-namespace/create-namespace: line 27: abs: command not found
namespace/bozo created
namespace/bozo labeled

KDEV1115:abbas> kubectl describe namespace bozo
Name:         bozo
Labels:       owner=nobody
Annotations:  <none>
Status:       Active
```

**Review by jarojasm95 — APPROVED** [review]
lgtm
